src/main/webapp/WEB-INF/templates/system/common/navigation.ftl

```diff
@@ -0,0 +1,166 @@
+<#include "/templates/system/common/ice.ftl"/>
+
+<#macro navigation
```

Comment:
We will need to document how to migrate from the previous implementation https://docs.craftercms.org/en/4.0/developers/projects/engine/api/templating-api.html#rendering-navigation